java/Ice/middleware/server/src/main/java/com/example/ice/middleware/server/AuthorizationMiddleware.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) ZeroC, Inc.
+
+package com.example.ice.middleware.server;
+
+import com.zeroc.Ice.DispatchException;
+import com.zeroc.Ice.IncomingRequest;
+import com.zeroc.Ice.Object;
```

Comment:
I feel it would be clearer to always qualify com.zeroc.Ice.Object to avoid confusion with java.lang.Object.